### DISCUSSION TITLE
Is there a way to configure email notifications to come from the right person?

### DISCUSSION THREAD

**Discussion by brendanheywood** [outline/outline#7424] [Ideas]
Currently all email comes from a generic outline user:

![image](https://github.com/user-attachments/assets/303ca369-cf24-4e00-93ab-27e6e414895e)

What I'm after is keeping the from email unchanged but overriding the human readable label in the From header to be from in this case Nicholas

Current state:

```
From: outline@catalyst-it.io
To: brendan@catalyst-au.net
Reply-To: outline@catalyst-it.io
```

Desired state:

```
From: Nicholas <outline@catalyst-it.io>
To: brendan@catalyst-au.net
Reply-To: outline@catalyst-it.io
```

Just about all other notification emails do this, eg from jira, gitlab, github, email lists etc. Here are example headers from a github issue notification:

```
From: Brendan Heywood <notifications@github.com>
Reply-To: catalyst/moodle-auth_outage <reply+AABNYOO6JWJU6YM6UDZDK6OED5MKLEVBNHHIJ3XH7Q@reply.github.com>
To: catalyst/moodle-auth_outage <moodle-auth_outage@noreply.github.com>
```

So is this possible with config or is this a feature request?

**Comment by tommoor** [comment]
This should already be the case, what version of the software are you running?

**Comment by brendanheywood** [comment]
thanks @tommoor for the quick response, we are running v0.78.0:

![image](https://github.com/user-attachments/assets/b6e9ee69-f958-46eb-bf14-a7549e2db512)

Is there any config we could be missing to affect this?

**Reply by tommoor** [reply]
There's no config, it's a small code fix – will be updated for `v0.79.0`

**Reply by brendanheywood** [reply]
thanks @tommoor :)

If it's not too much hassle, another low hanging fruit here is adding the threading headers so that multiple edits to the same document get threaded together:

If you have an example document like https://acme.com/doc/example-docs-hX4nZCk0ac then you can easily thread them like this:

```
In-Reply-To: <edit-hX4nZCk0ac@acme.com>
References: <edit-hX4nZCk0ac@acme.com>
Thread-Index: <edit-hX4nZCk0ac@acme.com>
```

I'm not sure if you'd want comment in a doc threaded with edits to a doc, they could be 2 sets of threads.

This way even if the document name changes they'll all get grouped together. The first two headers are the rfc headers, the second two add support for Outlook / Office365. Note that hX4nZCk0ac@acme.com is supposed to reference another message id but it doesn't need to reference a real one, all that is needed is for the related emails to reference the same id to group them. 

https://www.rfc-editor.org/rfc/rfc2822

**Reply by tommoor** [reply]
It would certainly be a good improvement, we add "Re:" to the subject line on comments which helps in threading those but probably not as reliably as these headers would achieve.

**Reply by brendanheywood** [reply]
Gmail does heuristic threading and ignore the headers and will pick up the 'Re:' but not thunderbird or outlook. It's a real mixed bag with support so gotta do it a few ways - thanks for making the issue and for the prompt response :)

**Comment by brendanheywood** [comment]
To clarify here I am talking about emails for when a document is edited. It appears that when a document is commented on then those emails do have the From header filled in.

**Comment by brendanheywood** [comment]
It occurs to me that perhaps this is because there could be a series of different edits by more than one person and these get grouped into a single email. 

If that is the case, could this be improved so that in the case where there was only one editor then it picks up their name? The majority of edits are single user edits
